taxi/views.py

```diff
@@ -28,6 +29,16 @@ def index(request):
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required
+def assign_driver(request, pk: int, assign: bool):
+    car = Car.objects.get(id=pk)
```

Comment:
Consider handling the case where the `Car` object with the given `id` does not exist. You can use `get_object_or_404` to return a 404 error if the object is not found.